bin/ycsb

```diff
@@ -140,9 +140,9 @@ def is_distribution():
 # presumes maven properly handles system-specific path separators
 def get_classpath_from_maven(database):
     try:
-        debug("Running 'mvn -pl com.yahoo.ycsb:"+database+"-binding -am package -DskipTests "
+        debug("Running 'mvn -pl com.yahoo.ycsb:"+database+" -am package -DskipTests "
               "dependency:build-classpath -DincludeScope=compile -Dmdep.outputFilterFile=true'")
-        mvn_output = subprocess.check_output(["mvn", "-pl", "com.yahoo.ycsb:"+database+"-binding",
+        mvn_output = subprocess.check_output(["mvn", "-pl", "com.yahoo.ycsb:" + database,
```

Comment:
won't this fail to build the correct module for any of the datastore specific bindings? e.g., if I request hbase10 the name of that module is 'hbase10-binding'